server/src/main/java/org/candlepin/guice/CandlepinModule.java

```diff
@@ -329,6 +338,25 @@ private void configureExporter() {
         bind(EntitlementCertExporter.class);
     }
 
+    private void configureSwagger() {
+        /**
+         * Using this binding, the swagger.(json|xml) will be available
+         * for an authenticated user at context: URL/candlepin/swagger.json
+         */
+        bind(ApiListingResource.class);
+        bind(SwaggerSerializers.class);
+
+        bind(CandlepinSwaggerModelConverter.class);
+
+        BeanConfig beanConfig = new BeanConfig();
+        beanConfig.setSchemes(new String[] { "http" });
```

Comment:
I see later down that we are requiring basic auth?  If so, we need to server over https.